wandb/apis/public.py

```diff
@@ -490,6 +490,33 @@ def _exec(self, query, **kwargs):
         variables.update(kwargs)
         return self.client.execute(query, variable_values=variables)
 
+    def _sampled_history(self, keys, x_axis="_step", samples=10000):
```

Comment:
samples=10000 ignored because history sets it, is it intended to be higher than samples=500 for some reason?